changelog/fragments/1724253791-replace-math-rand-with-math-rand-v2.yaml

```diff
@@ -0,0 +1,32 @@
+# Kind can be one of:
```

Comment:
I'm not completely sure if this needs a changelog. I saw that @pierrehilbert added a skip-changelog tag, that means I can remove this fragment or is it better to keep it?

Reply:
It means you can remove it.

Reply:
Thanks!